html/constants.js

```diff
@@ -0,0 +1,143 @@
+export const calories_per_day = 75,
```

Comment:
Whoops, my bad. I misread [this table](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#Browser_compatibility) believing I was looking at the compatibility table for the `import` statement. On a second look I figured out my slip.

That sorted out, one could leave `constants.js` untouched and bunch all imports in the other scripts into a `Constants` namespace:
```javascript
import * as Constants from './constants.js'
```

Reply:
Ah, yeah. It's easy to mix up tabs/features when trying to figure out browser compatibility, I've definitely done that a few times. I'll delay any additional cleanup tweaks until later, since everything else seems good.